### DISCUSSION TITLE
App Info

### DISCUSSION THREAD

**Discussion by btc8436954** [rainbow-me/rainbowkit#795] [Q&A] [answered]
<img width="356" alt="image" src="https://user-images.githubusercontent.com/49041460/193910118-cefa0e55-61a9-41e3-bfce-dded879e40d4.png">

Is it possible to remove the `learn more` section? Instead of changing the URL?

**Comment by peduarte** [comment, marked as answer]
hey! not currently. mainly because we believe the educational section is beneficial for new users. why do you want to hide it?

**Reply by btc8436954** [reply]
that makes sense. just wanted to make the UI as simple as possible. is it a violation of your TOS?

**Reply by peduarte** [reply]
no it's not. are you able to hide via css? 

we can consider an API for this though if there is enough demand, just cautious about bloating the API surface for now

**Reply by btc8436954** [reply]
cool. yeah, it is possible via css

**Reply by peduarte** [reply]
nice one!
